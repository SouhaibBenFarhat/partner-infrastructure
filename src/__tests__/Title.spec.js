import React from "react";
import {create} from "react-test-renderer";
import Title from "../App/Components/Charts/Title";

describe("Title component", () => {

    test("Matches the snapshot", () => {
        const title = create(<Title/>);
        expect(title.toJSON()).toMatchSnapshot();
    });

    test("Show the correct text", () => {
        const title = create(<Title>{'Random text'}</Title>);
        expect(title.toJSON().children[0]).toMatch('Random text')
    });
});

