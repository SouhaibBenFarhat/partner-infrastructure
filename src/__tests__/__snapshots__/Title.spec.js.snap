// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component Matches the snapshot 1`] = `
<h2
  className="MuiTypography-root MuiTypography-h6 MuiTypography-colorPrimary MuiTypography-gutterBottom"
/>
`;
